### PR TITLE
HW-5311. Added list Receipts

### PR DESCRIPTION
**Pull request by azakrevska-epam**

### CODE REVIEW

File: core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+        assertThat(receipt.getFee(), is("0.00"));
+        assertThat(receipt.getDetails(), is(notNullValue()));
+        final ReceiptDetails receiptDetail = receipt.getDetails();
+        assertThat(receiptDetail.getClientPaymentId(), is("8OxXefx5"));
+        assertThat(receiptDetail.getPayeeName(), is("A Person"));
+
+    }
+
+    @Test
+    public void testListReceipts_returnsCreditReceipt() throws InterruptedException {
```
Comment (fmattos-hw) [inline core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java]: I think we don't need this test. The test testListReceipts_returnsReceipts already checks it

File: core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import okhttp3.mockwebserver.RecordedRequest;
+
+@RunWith(RobolectricTestRunner.class)
+public class HyperwalletListReceiptsTest {
```
Comment (fmattos-hw) [inline core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java]: For all the tests, also verify that the http method was used. See HyperwalletListBankAccountsTest

File: core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java (hunk 1 above)
Comment (azakrevska-epam) [reply core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java]: @fmattos-hw Done

File: core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java (hunk 2 above)
Comment (azakrevska-epam) [reply core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListReceiptsTest.java]: @fmattos-hw added for other tests too.

**Pull request merged**